docs/develop.md

```diff
@@ -64,7 +64,7 @@ The easiest way to get developing is to use `kind` to bring up a cluster running
 1. Create the cluster:
 
    ```sh
-   kind create cluster --name=kpt-functions --config=kind.yaml --image=kindest/node:v1.14.6
+   kind create cluster --name=kpt-functions --config=kind.yaml --image=kindest/node:v1.15.9
```

Comment:
This instruction fails because of the image version, using kindest/node:v1.17.2 works for me.

Reply:
Changed to 1.15.7 to be closer to what is currently available in GCP.